tests/cdi/cdi_suite_test.go

```diff
@@ -0,0 +1,36 @@
+package cdi_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	ktests "kubevirt.io/kubevirt/tests"
+	_ "kubevirt.io/kubevirt/tests"
+)
+
+func TestCDI(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "CDI Suite")
+}
+
+var _ = BeforeSuite(func() {
+	out, err := ktests.RunOcCommand("get", "projects")
+	Expect(err).ToNot(HaveOccurred())
+	Expect(out).Should(ContainSubstring("golden-images"))
+
+	_, err = ktests.RunOcCommand("project", "golden-images")
+	Expect(err).ToNot(HaveOccurred())
+})
+
+var _ = AfterSuite(func() {
+	_, err := ktests.RunOcCommand("delete", "pvc", pvcName)
```

Comment:
I don't see where `pvcName` is defined in scope to be referenced here, but since this runs I guess it works. From my perspective it's cleaner to define the variables referenced in `BeforeSuites` and `AfterSuites` in the __suite.go_ files.